tests/recovery/recoverySecurity.test.ts

```
@@ -371,5 +371,81 @@ describe('Recovery Security E2E Tests', () => {
 
       await expect(verifyRecoveryOtp(verifyParams)).rejects.toThrow('Invalid code');
     });
+
+    it('should lock the OTP challenge after 5 wrong guesses (BE-018)', async () => {
```

ESLint

[error] Replace `'should·lock·the·OTP·challenge·after·5·wrong·guesses·(BE-018)'` with `"should·lock·the·OTP·challenge·after·5·wrong·guesses·(BE-018)"`

(prettier/prettier)